apps/docs/src/common/pages/get-started/project.tsx

```diff
@@ -0,0 +1,115 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
+
+
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <div className="govuk-grid-column-one-third">
+      <aside>
+        <h2>Get started</h2>
+          <ul className="plain">
+            <li><A href="/get-started/prototype">Prototyping for designers</A></li>
+            <li><A href="/get-started/project">Starting a project for developers</A></li>
+          </ul>
+      </aside>
+    </div>
+
+    <div className="govuk-grid-column-two-thirds">
+      <h1>
+        <span className="caption">Get started</span>
+        Starting a project for developers
+      </h1>
+        <p>This guide explains how to set up a project in production using the
+        coded React examples from the Home Office Design System.</p>
+
+        <p>In order to consume these components you will require a system that utilised a bundler (such as <A href="https://webpack.js.org/">Webpack</A>) that can process imported assets such as images, fonts and <A href="https://sass-lang.com/">Sass</A> files. <A href="https://create-react-app.dev/">Create React App</A> may be able to do this but does not provide Server-Side Rendering (SSR). You will also need to ensure that you provide an instance of <A href="https://reactrouter.com/">react-router</A>.</p>
+
+        <p>As such, we suggest that you use our specially designed tech stack for this purpose. You can start a brand new project or prototype using the following steps.</p>
```

Comment:
This needs to be moved back into the main 'get started' page. It is useful to people who just want to consume the components rather than use the full framework.

Reply:
> Looks like a really good start.
> 
> I do wonder if we should have Markdown rather than HTML though as that way people can read the docs from GitHub. (Something might devs like.)
> 
> I can sort that out though.

Happy for you to go ahead and have it in Markdown although may be easier for you to set it up as I'm unsure how to do it myself. In the meantime, I've updated the latest branch with all the comments you've made if you want to review this for the Markdown